libtcmu_log.c

```diff
@@ -63,6 +60,36 @@ struct log_output {
 	tcmu_log_destination dest;
 };
 
+/* tcmu log dir path */
+static char *tcmu_log_dir = NULL;
+
+/* get the log dir of tcmu-runner */
+char *tcmu_get_log_dir(void)
+{
+	return tcmu_log_dir;
+}
+
+char *tcmu_alloc_and_set_log_dir(const char *log_dir)
+{
+	/*
+	 * Do nothing here and will use the /var/log/
+	 * as the default log dir
+	 */
+	if (!log_dir)
+		return NULL;
+
+	tcmu_log_dir = strdup(log_dir);
+	if (!tcmu_log_dir)
+		tcmu_err("Failed to copy log dir: %s\n", log_dir);
+
+	return tcmu_log_dir;
+}
+
+void tcmu_free_log_dir(void)
+{
```

Comment:
Just name this tcmu_destroy_log_dir. It then matches the create/destroy balancing.

Reply:
Sure